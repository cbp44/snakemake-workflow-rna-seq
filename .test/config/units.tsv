sample	unit	fq1	fq2	encode_biosample	encode_library
bclaf1_kd_rep1	1	ENCSR410ZPU_L01_1_1.fq.gz	ENCSR410ZPU_L01_1_2.fq.gz	ENCBS468YAY	ENCLB854NIY
bclaf1_kd_rep2	1	ENCSR410ZPU_L01_2_1.fq.gz	ENCSR410ZPU_L01_2_2.fq.gz	ENCBS417QMS	ENCLB272YKU
ctrl_rep1	1	ENCSR245BNJ_L01_1_1.fq.gz	ENCSR245BNJ_L01_1_2.fq.gz	ENCBS661YYO	ENCLB111QIC
ctrl_rep2	1	ENCSR245BNJ_L01_2_1.fq.gz	ENCSR245BNJ_L01_2_2.fq.gz	ENCBS250HLF	ENCLB775RZU
