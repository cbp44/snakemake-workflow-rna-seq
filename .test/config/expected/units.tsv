sample	unit	fq1	fq2
bclaf1_kd_rep1	1	ENCSR410ZPU_L01_1_1.fq.gz	ENCSR410ZPU_L01_1_2.fq.gz
bclaf1_kd_rep2	1	ENCSR410ZPU_L01_2_1.fq.gz	ENCSR410ZPU_L01_2_2.fq.gz
ctrl_rep1	1	ENCSR245BNJ_L01_1_1.fq.gz	ENCSR245BNJ_L01_1_2.fq.gz
ctrl_rep2	1	ENCSR245BNJ_L01_2_1.fq.gz	ENCSR245BNJ_L01_2_2.fq.gz